src/components/Avatars/Avatar/Avatar.styled.ts

```diff
@@ -15,10 +16,16 @@ interface AvatarProps extends InterpolationProps {
   showBorder: boolean
 }
 
-function calcStatusGap(showBorder: boolean) {
-  return `${showBorder && enableSmoothCorners.current
-    ? (AVATAR_BORDER_WIDTH * 2) - AVATAR_STATUS_GAP
-    : -AVATAR_STATUS_GAP}px`
+interface StatusWrapperProps extends Pick<AvatarProps, 'showBorder'> {
+  size: StatusSize
+}
+
+function calcStatusGap(props: StatusWrapperProps) {
+  let size = (props.size >= StatusSize.L ? 4 : -2)
+  if (props.showBorder && enableSmoothCorners.current) {
+    size += AVATAR_BORDER_WIDTH * 2
+  }
+  return size
```

Comment:
```suggestion
function calcStatusGap({ size, showBorder }: StatusWrapperProps) {
  let gap = (size >= StatusSize.L ? 4 : -2)
  if (showBorder && enableSmoothCorners.current) {
    gap += AVATAR_BORDER_WIDTH * 2
  }
  return gap
```

구조 분해 할당 구문을 사용하면 더 깔끔할 거 같고, size 보다 gap이 더 의미가 잘 드러날 거 같아요.